README.md

```diff
@@ -154,7 +175,7 @@ How many valid sentences are there in this language?
 
 ## A2.6
 
-«replace this with your answer»
+48
```

Comment:
# 0

There are an infinite number, as you can do the S3 -> S3 transition for as long as there's an adjective in the input.